scripts/generate_sitemap.py

```diff
@@ -160,10 +187,14 @@ def generate(self):
         # TODO: server side cursor query wrapper might be useful as the index gets larger.
         # Sitemap indexable limit check
         if self.sitemap_count > settings.SITEMAP_INDEX_MAX * .90: # 10% of urls remaining
-            logger.exception('WARNING: Max sitemaps nearly reached: {} of 50000 max sitemaps for index file'.format(str(self.sitemap_count)))
-            log_exception()
+            sentry.log_message('WARNING: Max sitemaps nearly reached: {} of 50000 max sitemaps for index file'.format(str(self.sitemap_count)))
         print('Total url_count = {}'.format((self.sitemap_count - 1) * settings.SITEMAP_URL_MAX + self.url_count))
         print('Total sitemap_count = {}'.format(str(self.sitemap_count)))
+        if self.errors:
+            sentry.log_message('WARNING: Generate sitemap encountered {} error(s). See logs for details.'.format(self.errors))
```

Comment:
The title of the sentry message for this should be static so that it doesn't create a new event each time. The details can differ, but the titles need to be the same. Also, this one should be lower priority than the one above where we had to cancel building the sitemap.